### PR TITLE
[C#] Add code syntax samples to numbers-floating-point exercise

### PR DESCRIPTION
**Pull request by ErikSchierboom**
See #986.

### CODE REVIEW

File: languages/csharp/exercises/concept/numbers-floating-point/.docs/introduction.md [hunk 1]
```diff
@@ -10,4 +10,11 @@ C# has three floating-point types:
 
 As can be seen, each type can store a different number of digits. This means that trying to store PI in a `float` will only store the first 6 to 9 digits (with the last digit being rounded).
 
-In this exercise you may also want to use a loop. There are several ways to write loops in C#, but the `while` loop is most appropriate here.
+In this exercise you may also want to use a loop. There are several ways to write loops in C#, but the `while` loop is most appropriate here:
+
+```csharp
+while (x > 10)
```
Comment (wolf99) [inline languages/csharp/exercises/concept/numbers-floating-point/.docs/introduction.md]: Should syntax examples be minimally working? i.e. in this case, should there be included a new first line of `int x = 0` ?

Comment (ErikSchierboom) [reply languages/csharp/exercises/concept/numbers-floating-point/.docs/introduction.md]: Good idea. I've added a variable. 

The syntax example should not necessarily be minimally working.

**Comment by wolf99** [comment]
Change looks good @ErikSchierboom 🙂 
Slightly wider question though: should while loops be a prerequisite for this exercise?

**Comment by ErikSchierboom** [comment]
> Slightly wider question though: should while loops be a prerequisite for this exercise?

No, this exercise _introduces_ while loops :) And thanks for the review!

**Pull request merged**